### DISCUSSION TITLE
Specify grain size

### DISCUSSION THREAD

**Discussion by Sofia-Y** [neperfepx/neper#214] [Q&A] [answered]
This question might seem very simple: can we specify grain sizes using Neper? (example 125 um?)

Thanks in advance!

**Comment by rquey** [comment, marked as answer]
Use `-n from_morpho -morpho "diameq:lognormal(0.125,...),1-sphericity:lognormal(0.0855,0.03)"` for a lognormal grain size distribution of average 0.125 (and controlled sphericity). Dimensions in Neper are arbitrary, so you can write 0.125 for millimeters of 0.000125 for meters, You will need to set the size of the domain accordingly (`-domain "cube(...,...,...)"`).

Example:
```
$ neper -T -n from_morpho  -morpho "diameq:lognormal(0.125,0.04),1-sphericity:lognormal(0.145,0.03)" -domain "cube(0.5,0.5,0.5)"
```
